src/options/view.rs

```diff
@@ -458,23 +458,6 @@ mod test {
             }
         };
 
-        ($name:ident: $type:ident <- $inputs:expr; $stricts:expr => like $pat:pat) => {
-            /// More general macro for testing against a pattern.
-            /// Instead of using `PartialEq`, this just tests if it matches a pat.
-            #[test]
-            fn $name() {
-                for result in parse_for_test($inputs.as_ref(), TEST_ARGS, $stricts, |mf| {
-                    $type::deduce(mf)
-                }) {
-                    println!("Testing {:?}", result);
-                    match result {
-                        $pat => assert!(true),
-                        _ => assert!(false),
-                    }
-                }
-            }
-        };
-
```

Comment:
Instead of deleting this, can we make up some test that covers it so it isn't an issue, it might come in handy later?

Reply:
So I guess adding that like here would work https://github.com/eza-community/eza/blob/0b006974e9ea486340f69a159fbc216845b94fcc/src/options/view.rs#L434?